userspace/libsinsp/sinsp.h

```diff
@@ -89,14 +80,21 @@ limitations under the License.
 #include <libsinsp/tuples.h>
 #include <libsinsp/fdinfo.h>
 #include <libsinsp/threadinfo.h>
-#include <libsinsp/ifinfo.h>
 #include <libsinsp/eventformatter.h>
-
-#include <libsinsp/include/sinsp_external_processor.h>
```

Comment:
Apart from some re-ordering of the includes to put the standard ones at the end, the real change here is the removal of this `libsinsp/include/...` layer.